content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -61,6 +61,11 @@ If `startingDeadlineSeconds` is set to a large value or left unset (the default)
 and if `concurrencyPolicy` is set to `Allow`, the jobs will always run
 at least once.
 
+{{< caution >}}
+If `startingDeadlineSeconds` is set to a value less than 10 seconds, the CronJob may not be scheduled. This is because the CronJob controller checks things every 10 seconds.
```

Comment:
It might be worth mentioning that _as an alpha feature_ there is a V2 controller implementation with different behavior.

Reply:
@sftim I can see a new CronJob v2alpha1 version on the API reference but I can't see what are the changes. Can you point me to a document or something where I can see the new changes?

Thanks.

Reply:
The draft documentation for Kubernetes v1.20 mentions the feature gate for the v2 CronJob controller: https://deploy-preview-24138--kubernetes-io-master-staging.netlify.app/docs/reference/command-line-tools-reference/feature-gates/#feature-gates